### ISSUE TITLE
Push to live site

### ISSUE THREAD

**Issue by lollypop27** [CraftAcademy/slow_food_june#28]
Setup & Push to live site as soon as possible

**Comment by tochman** [comment]
1. Assign an owner of this story
   This person will have admin access to the Heroku instance and thus control of the deploy. She or He can add others to become admins later but for now let's stick with one person being responsible for deployment.
2. Create a Heroku account if you don't have one already
   The first thing you need to do is to head over to http://heroku.com and set up a new account. Make sure that you remember your username and password, as you will need them in a short while.
3. Install Heroku CLI
   Next, head over to toolbelt.heroku.com/ and download the Heroku Toolbelt, a command line interface tool (CLI) for creating and managing Heroku apps.
4. Login to Heroku from your terminal
   Once installed, you can use the heroku command from your command shell. Log in using the email address and password you used when creating your Heroku account (this authentication is required to allow both the heroku and git commands to operate)
   
   ```
   $ heroku login
   Enter your Heroku credentials.
   Email: 
   Password: 
   ...
   ```
5. create a Heroku application
   
   ```
   $ heroku create sf-sinatra # choose your own name
   Creating sf-sinatra... done, stack is cedar-14
   https://sf-sinatra.herokuapp.com/ | https://git.heroku.com/sf-sinatra.git
   Git remote heroku added
   ```
6. Run the heroku console
   Basically google it ;-) 
7. Create the database
   Again, some googling on how to add the PG add-on and then create the database (perhaps not needed, figure it out.)
8. Open the app in your browser
   
   ```
   heroku open
   ```

**Comment by lollypop27** [comment]
www.heroku.com
user: lara.t.walters@gmail.com
pw: craft123!

**Comment by lollypop27** [comment]
https://slow-food-june.herokuapp.com

https://git.heroku.com/slow-food-june.git

**Comment by lollypop27** [comment]
I have set Heroku to deploy when a push is made to master and that the code has passed.
